### ISSUE TITLE
Bug: BitmapTarget::copy_raw_pixels ends the draw on D2D

### ISSUE THREAD

**Issue by CryZe** [linebender/piet#408]
https://github.com/linebender/piet/blob/4268b616fd292e0f80068ad87ab5206cf16ac4f8/piet-common/src/direct2d_back.rs#L165

Here the draw ends, which means the whole BitmapTarget becomes unusable afterwards and causes the application to crash if you use its `render_context()` again:

```
D2D DEBUG ERROR - An attempt to render a primitive outside of BeginDraw/EndDraw.
```

**Comment by cmyr** [comment]
Ah, okay; this method used to be called `into_raw_pixels`, and it used to consume the `BitmapTarget`. Although d2d remains a bit of a mystery to me I think we should be fine to remove this, and rely on `drop` to call `end_draw`.

**Comment by longmathemagician** [comment]
This is fixed by #569 & #578.
